blockware/fireworks/src/main.cpp

```diff
@@ -256,6 +256,15 @@ void updateWakeState()
   }
 }
 
+void updateGravity()
+{
+  // Read accelerometer
+  int16_t axes[3];
+  acc->Get_X_AxesRaw(axes);
```

Comment:
Not sure if it's a meaningful performance impact but this causes us to do `Get_X_AxesRaw` twice per tick which is a blocking SPI transaction I think. Maybe worth refactoring out so that `Get_X_AxesRaw` happens from `main` and then gets passed into both `updateGravity` and `updateWakeState`

Reply:
Thanks!  I also wasn't sure if it's a performance hit- what do you think of this approach?

Reply:
Looks good to me!